templates/field/field--oe-contact--oe-link--full.html.twig

```diff
@@ -0,0 +1,21 @@
+{#
+/**
+ * @file
+ * Link field in the Contact entity default view mode.
+ *
+ * @see ./core/themes/stable/templates/field/field.html.twig
+ */
+#}
+{% for item in items %}
+  <div class="ecl-u-border-bottom ecl-u-border-color-grey-15 ecl-u-pt-l ecl-u-pb-l">
+    {% include '@ecl-twig/link' with {
+      link: {
+        type: 'standalone',
+        label: item.content['#title'],
+        path: item.content['#url'],
+        icon_position: 'after'
+      },
+      icon: get_link_icon(item.content['#url'].toString())
```

Comment:
By accessing link render array portions we don't allow cache to bubble up. The easiest way to fix it is to render the link and assign the output to a catch all variable, that will never be displayed, like that:

```twig
   {% set bubble_cache = item.content|render %}
```

Check how it was done in [this PR](https://github.com/openeuropa/oe_theme/pull/787/files#diff-e8ef914d08dc95ae7d28b49062142ef7e0181ea1dff787b83d2a5cbd5cf7a6f1R10).

Reply:
Added